flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -171,6 +171,11 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
 
     val appStartMetrics = AppStartMetrics.getInstance()
 
+    if (!appStartMetrics.isAppLaunchedInForeground) {
+      Log.w("Sentry", "App not launched in Foreground")
```

Comment:
```suggestion
      Log.w("Sentry", "Invalid app start data: app not launched in foreground")
```

let's be a bit more specific here